src/redux/admin/admin.operations.js

```diff
@@ -0,0 +1,57 @@
+import { gql } from 'apollo-boost';
+import { client } from '../../utils/client';
+
+export const loginAdmin = async (user) => {
+  const result = await client
```

Comment:
Please, implement operation in the same way as it was done in your another merge request, it is quite good approach